Accessibility/VirtualContent/explainer.md

```diff
@@ -129,6 +129,72 @@ window.addEventListener("scroll", on_scroll_changed);
 13. Eventually, the backing server sends a content payload which also contains a signal that the end of the document has been reached. Script code removes the `aria-virtualcontent` attribute from the `main` element to indicate there is no more virtualized content.
 14. The next time the user attempts to navigate to the next heading, the AT searches for a virtual content container, finds none, and announces there are no further headings in the document.
 
+## Example Use 2: Tabular data
+
+This example illustrates one use of `aria-virtualcontent` for virtualized content in the inline direction. The document in this scenario is a table of bug reports with the following columns: report ID, report date, status, assigned to, title, fix date, fixed by. The last two columns have been virtualized out of the DOM based on the viewport width. When realized, these two columns do not have any data for bugs that are not yet fixed.
+
+```
+<html>
+<body>
+<table aria-virtualcontent="block-end">
+  <thead>
+    <tr aria-virtualcontent="inline-end">
+      <th>ID</th>
+      <th>Status</th>
+      <th>Assigned To</th>
+      <th>Report Date</th>
+      <th>Title</th>
```

Comment:
Need something in the markup of the example to reiterate that the last 2 columns are virtualized. Perhaps comment out the virtualized cells with a note.

Reply:
Done